<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DimensionSliceWidgetClass</class>
 <widget class="QWidget" name="DimensionSliceWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DimensionSliceWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Dim.Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnX">
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Check to make this dimension X in the 2D view</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnY">
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Check to make this dimension Y in the 2D view</string>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
   </item>
   <item>
    <widget class="QLabel" name="lblUnits">
     <property name="text">
      <string>Ang</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
